ManatiCR/vitaminau
# VIT-72: Accessibility

# Task [VIT-72](https://manati.atlassian.net/browse/VIT-72)
Fixes to improve accessibility.

# Installation
- `ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the site at the following URL and check that the pages have a score of 100 for accessibility:
- Start the accessibility test with "lighthouse".
  - http://vitaminau.docker/user
  - http://vitaminau.docker/formularios
Note: remember that the forms will not have a 100 since the "select2" is kept.
  - http://vitaminau.docker/formularios/apoyo-al-emprendimiento
  - http://vitaminau.docker/formularios/practica-profesional
  - http://vitaminau.docker/formularios/servicios-tecnicos
  - http://vitaminau.docker/formularios/tecnologias-licenciables
  - http://vitaminau.docker/formularios/trabajo-comunal-universitario
  - http://vitaminau.docker/formularios/trabajo-en-curso
  - http://vitaminau.docker/formularios/uso-de-instalaciones
  - http://vitaminau.docker/formularios/venta-de-productos

# CA
- Verify that the pages have resolved accessibility issues.
